modules/storages/app/models/storages/storage.rb

```diff
@@ -119,6 +119,19 @@ def self.extract_part_from_piped_string(text, index)
       end
     end
 
+    def oauth_access_granted?(user)
+      selector = Peripherals::StorageInteraction::AuthenticationMethodSelector.new(
+        storage: self,
+        user:
+      )
+      case selector.authentication_method
+      when :sso
+        true
+      when :storage_oauth
+        OAuthClientToken.exists?(user:, oauth_client: oauth_client)
```

Comment:
:green_circle: We could use short syntax here

```suggestion
        OAuthClientToken.exists?(user:, oauth_client:)
```